next/components/layout/layout.tsx

```diff
@@ -128,20 +128,35 @@ export default function Layout({
                                                             >
                                                                 <SideNavGroup level={3}>
                                                                     {groupItem.sections?.map(
-                                                                        sectionItem => (
-                                                                            <SideNavLink
-                                                                                title={
-                                                                                    sectionItem.title
-                                                                                }
-                                                                                href={
-                                                                                    sectionItem.href
-                                                                                }
-                                                                                key={
-                                                                                    sectionItem.href
-                                                                                }
-                                                                                level={3}
-                                                                            />
-                                                                        ),
+                                                                        sectionItem => {
```

Comment:
This is getting heavily nested to the point it would be better to extract into its own component

Reply:
Hah yeah, very fair. This component is unwieldy.